crawler/reuters.py

```diff
@@ -49,14 +49,17 @@ def fetch_content(self, task, date_range):
         ticker, name, exchange, market_cap = task
         print("%s - %s - %s - %s" % (ticker, name, exchange, market_cap))
 
-        suffix = {'AMEX': '.A', 'NASDAQ': '.O', 'NYSE': '.N'}
-        # e.g. https://www.reuters.com/finance/stocks/company-news/BIDU.O?date=09262017
-        url = "https://www.reuters.com/finance/stocks/company-news/" + ticker + suffix[exchange]
+        suffix = {'AMEX': '.A', 'NASDAQ': '.O', 'NYSE': '.N', 'LSE': '.L', 'PSE': '.PA'}
+        # e.g. https://www.reuters.com/companies/BIDU.O/news
+        # legacy e.g. https://www.reuters.com/finance/stocks/company-news/BIDU.O?date=09262017
+        url = "https://www.reuters.com/companies/" + ticker + suffix[exchange] + "/news"
```

Comment:
Refactor out ticker and exchange matching - I don't see the benefit. The golden source here is the ticker itself, and of we need to say what exchange a stock is on, we can determine that from the ticker.

This causes issues because parsed news articles are given tickers in the form:

```
ACCP
AV
ENQ
EVEE
EZJ
GYM
LLOY
NSCI
```

These don't match with Yahoo Finance tickers, which also uses the suffix.

Furthermore - Reuters tickers don't always match Yahoo Finance tickers. e.g. ACCP.PA is AC.PA on Yahoo Finance (verify this).